pages/index.vue

```diff
@@ -2,6 +2,41 @@
 
     <b-container fluid>
 
+        <b-row align-h="center">
+            <b-col cols="6">
+                <h1>
+                    👋 Welcome to the neurobagel annotation tool
+                </h1>
+                <p>
+                    This tool allows you to create a machine readable data dictionary
+                    in .json format for a tabular phenotypic file in .tsv format (Data table).
+                    If you already have a BIDS-like .json data dictionary, you can provide it here
+                    and augment it with more detailed annotations.
+                </p>
+                <p>
+                    🏗️ This tool (like all of neuroabgel) is under active development and we are thankful
+                    for all feedback and suggestions you may have. You can either use our feedback widget
+                    on the side of this window, or open an issue on <a href="https://github.com/neurobagel/annotation_tool" target="_blank">our GitHub repository</a>.
+                </p>
+                <p>
+                    The process of annotations works like this:
+                </p>
+                <ol>
+                    <li>Upload a .tsv tabular phenotypic file (<a href="https://neurobagel.org/annotation_tool/" target="_blank">see our docs for details</a>)</li>
+                    <li>Upload a <a href="https://neurobagel.org/dictionaries/" target="_blank">BIDS-like .json data dictionary</a> (optional)</li>
```

Comment:
```suggestion
                    <li>Upload a <a href="https://neurobagel.org/dictionaries/" target="_blank">BIDS-compatible JSON data dictionary</a> (optional)</li>
```

I would also move the https://neurobagel.org/dictionaries/ link to a parenthetical like `(see our docs for more details on Neurobagel augmentation of BIDS data dictionaries)` since that page is more describing the **annotated** data dictionary rather than the BIDS one, if that makes sense